### ISSUE TITLE
Add information about what COPCSE-NTNU is to repo/org

### ISSUE THREAD

**Issue by odinuge** [COPCSE-NTNU/thesis-NTNU#27]
It would be nice to add a proper disclosure about "who" is behind this repo, if this is an "official" template, and what COPCSE is. I really like this, but when I first found it, I thought this were some random repo/org with NTNU in the title. Adding a proper disclosure about who are standing behind this repo would be nice in order to give it credability, and simply to show what this is. I still don't really understand if this is endorsed as official, or if some professors are saying that without it being the case. I think a better description would solve this.

This information (copy from the result PDF) is kinda ok, but it is hidden deep inside the repo.
```
Over the years, several thesis templates for LATEX have been developed by different groups at NTNU. Typically, there have been local templates for given
study programmes, or different templates for the different study levels – bachelor, master, and PhD.
Based on this experience, the Community of Practice in Computer ScienceEducation at NTNU (CoPCSE@NTNU)
is hereby offering a template
that should in principle be applicable for theses at all study levels. It is closely
based on the standard LATEX report document class as well as previous thesis
templates. Since the central regulations for thesis design have been relaxed –
at least for some of the historical university colleges now part of NTNU – the
template has been simlified and put closer to the default LATEX look and feel.
The purpose of the present document is threefold. It should serve (i) as a
description of the document class, (ii) as an example of how to use it, and (iii)
as a thesis template
```

Thanks for the work btw.

**Comment by ifarup** [comment]
CoPCSE is an informal forum for lecturers in computer science and related fields at NTNU across campuses and departments. The template does not have any official status, and it is not a general NTNU-level requirement to use it. If some study programme make it a 'local requirement', the students of that programme should be notified directly. It is simply provided as an offering/help for students writing their thesis (at any level: BSc, MSc, PhD). I will update the README to make this more clear.

**Comment by odinuge** [comment]
Thanks @ifarup!

My advisor said she had "heard" there were an official template, and pointed us at this. I found out what CoPCSE was after some google-ing, but it still took some time to verify that this was the same as I found on the webpage. I still think it will be very useful for others as well! :smile: 

Would be nice to update this page as well, to make it more clear what CoPCSE is, and what the "intentions" behind this account/org is.
![Screenshot from 2021-02-24 09-26-34](https://user-images.githubusercontent.com/1467188/108971558-b934bf80-7682-11eb-8128-13ae32a8fefd.png)

Again, thanks! :smile:
